Source/AliveLib/Slig.hpp

```diff
@@ -623,3 +565,8 @@ class Slig : public BaseAliveGameObject
 ALIVE_ASSERT_SIZEOF(Slig, 0x298);
 
 EXPORT int CC Animation_OnFrame_Slig_4C0600(void* pObj, signed __int16* pData);
+
+
+void renderWithGlowingEyes(int** ot, BaseAliveGameObject* actor, __int16* pPalAlloc, __int16 palSize, PSX_RECT* palRect,
+    __int16& r, __int16& g, __int16& b,
+    __int16 eyeColourIndices[]);
```

Comment:
should probably just be __int16 eyeColourIndices* size there is no size so it really is just a pointer

Reply:
oh and probably it should be const